src/features/gallery/components/PurchaseModal.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  Button,
+  FormControl,
+  FormErrorMessage,
+  FormHelperText,
+  Input,
+  Modal,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalOverlay,
+  Text,
+} from '@chakra-ui/react';
+import { Form } from 'react-router-dom';
+
+import { TicketIncrementer } from './TicketIncrementer';
+
+interface PurchaseModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  event: any;
+  onSubmit: () => void;
+  setEmail: (email: string) => void;
+  email: string;
+  setTicketAmount: (ticketAmount: number) => void;
+  amount: number;
+  decrementAmount: () => void;
+  incrementAmount: () => void;
+}
+
+export const PurchaseModal = ({
+  isOpen,
+  onClose,
+  event,
+  onSubmit,
+  setEmail,
+  email,
+  decrementAmount,
+  incrementAmount,
+  amount,
+}: PurchaseModalProps) => {
+  // email input
+  const handleInputChange = (e) => {
+    setEmail(e.target.value);
+  };
+  const isError = email === '';
+  return (
+    <Modal isCentered closeOnOverlayClick={false} isOpen={isOpen} size={'xl'} onClose={onClose}>
+      <ModalOverlay />
```

Comment:
Please apply this fix to all your modal components because it fixes the lag issue
```suggestion
            <ModalOverlay backdropFilter="blur(0px)" bg="blackAlpha.600" opacity="1" />
```